package.json

```diff
@@ -68,6 +75,7 @@
     "phantomjs": "^1.9.17",
     "react-hot-loader": "^1.2.8",
     "react-router": "^1.0.0-rc1",
+    "rimraf": "^2.4.3",
```

Comment:
Why is it a peerDependency and not a dependency?
I feel like this is not link to react v0.14 and should be added to `master`